src/app_service/service/ens/service.nim

```diff
@@ -0,0 +1,249 @@
+import NimQml, Tables, sets, json, sequtils, strutils, strformat, chronicles
```

Comment:
ENS module has been totally refactored in master, I would suggest to take that one, this is totally different now as we interact directly with status-go

Reply:
uhhh didn't know that... will take a look soon. Thank you.